README.md

```diff
@@ -118,7 +118,7 @@ kubectl delete -n conformance pods/e2e-conformance-test && kubectl delete ns con
 Please reach out for bugs, feature requests, and other issues!
 The maintainers of this project are reachable via:
 
-- [Kubernetes Slack] in the [#sig-testing] and [#k8s-conformance] channels
+- [Hydrophone Slack] in the [#sig-testing] and [#k8s-conformance] channels
```

Comment:
I'd like to keep the Kubernetes Slack link as is since that's the overarching slack community and will guide folks on getting into slack if they've not logged in before. 

We should add #hydrophone similar to how we list #sig-testing and #k8s-conformance.